corehq/apps/reports/views.py

```diff
@@ -384,6 +384,8 @@ def hq_download_saved_export(req, domain, export_id):
     # quasi-security hack: the first key of the index is always assumed
     # to be the domain
     assert domain == export.configuration.index[0]
+    export.last_accessed = datetime.utcnow()
```

Comment:
I think the main argument for doing it is excel dashboards (which may not ever see the UI)